ScutiSDK.asmdef

```diff
@@ -2,6 +2,7 @@
     "name": "Scuti",
     "references": [
         "Unity.TextMeshPro",
+        "Unity.InputSystem",
```

Comment:
@juanesdavinci will this still work/compile in a 2018.4 project? (Test by creating a 2018.4 project and input scuti and see if it works)